docs/root/intro/version_history.rst

```diff
@@ -52,6 +52,7 @@ Version history
   certificate validation context.
 * tracing: added tags for gRPC response status and meesage.
 * upstream: added :ref:`an option <envoy_api_field_Cluster.CommonLbConfig.close_connections_on_host_set_change>` that allows draining HTTP, TCP connection pools on cluster membership change.
+* tracing: added :ref:`max_path_tag_length <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.tracing>` to support customizing the length of the request path included in the extracted HttpUrl tag.
```

Comment:
Nit: you might want to do a `:ref:` back to `HttpUrl` message definition as well.

Reply:
added a pointer to the external definition of the tag. that ok?